cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PackCommand.java

```diff
@@ -246,6 +246,18 @@ public void execute() {
         // Check package files are modified after last build
         boolean isPackageModified = isProjectUpdated(project);
 
+        // Checks if Package.md is present and issues a warning
+        Path packageMd = project.sourceRoot().resolve(ProjectConstants.PACKAGE_MD_FILE_NAME);
+        if (packageMd.toFile().exists()) {
+            String warning = """
+
+                    WARNING: usage of Module.md and Package.md are deprecated.
```

Comment:
The use of Package.md and Module.md is deprecated. Update the package to add a README.md file.